pulp_ansible/app/viewsets.py

```diff
@@ -105,39 +103,9 @@ def filter_by_q(self, queryset, name, value):
         )
         return qs.annotate(rank=ts_rank_fn).order_by("-rank")
 
-    def filter_latest(self, queryset, name, value):
-        """
-        If the value of 'latest' is True, include only the latest Collection version in the results.
-
-        Args:
-            queryset: The already-formed queryset for modification
-            name: The name of the parameter, 'latest'
-            value: The value of the argument. This is checked if 'True' or not.
-
-        Returns:
-            Queryset with latest collections included if value is True.
-
-        """
-        if not value:
-            return queryset
-
-        namespace_name_dict = defaultdict(lambda: defaultdict(list))
-        for collection in queryset.all():
-            version_entry = (parse(collection.version), collection.pk)
-            namespace_name_dict[collection.namespace][collection.name].append(version_entry)
-
-        latest_pks = []
-        for namespace, name_dict in namespace_name_dict.items():
-            for name, version_list in name_dict.items():
-                version_list.sort(reverse=True)
-                latest_pk = version_list[0][1]
-                latest_pks.append(latest_pk)
-
-        return queryset.filter(pk__in=latest_pks)
-
     class Meta:
         model = CollectionVersion
-        fields = ["namespace", "name", "version"]
+        fields = ["namespace", "name", "version", "is_highest"]
```

Comment:
Should 'q' be in here too? What happens to CollectionVersion filtering if `fields` isn't specified at all?

Reply:
I'm new to Django, and `q` wasn't there before, I don't know why!